userspace/libscap/engine/gvisor/gvisor.cpp

```diff
@@ -159,6 +169,8 @@ extern const struct scap_vtable scap_gvisor_engine = {
 	.stop_capture = gvisor_stop_capture,
 	.configure = gvisor_configure,
 	.get_stats = gvisor_get_stats,
+	.get_stats_size_hint = NULL,
```

Comment:
@gnosek and @Andreagit97 having some trouble with CPP integration in general, could use some help. 

Re `gvisor` error is:
```
error: invalid conversion from ‘std::size_t (*)(scap_engine_handle, scap_stats*)’ {aka ‘long unsigned int (*)(scap_engine_handle, scap_stats*)’} to ‘size_t (*)()’ {aka ‘long unsigned int (*)()’} [-fpermissive]
  174 |         .get_stats_size_hint = gvisor_get_stats_size_hint,
```
In the PR it's currently `size_t`, but I also tried with some more includes and `std::size_t` just not having any luck with `gvisor_get_stats_size_hint` at all ... ideas on how to fix it or should I change the type?

In addition, started the sinsp integration and again while I managed getting over the `size_t` issues there via `std::size_t` passing the buffer array pointer gave errors which I think in C I just saw warnings. How should we pass stats instead?

```
cannot convert ‘scap_stats_v2 (*)[buf_size]’ to ‘scap_stats_v2*’ inspector.get_capture_stats_v2(buf_size, flags, &stats_v2);
```